.circleci/config.yml

```diff
@@ -46,158 +42,113 @@ jobs:
+      CLOUDIFY_SSL_TRUST_ALL: true
+      IAAS: aws
+    steps:
+      - checkout
+      - attach_workspace:
+          at: workspace
       - run:
-          name: install plugins bundle
-          description: The user will need to upload all plugins.
+          name: "Pull Submodules"
           command: |
-            cfy plugins bundle-upload
-            for plugin in `cfy executions list --include-system-workflows | grep install_plugin | awk '{print $2}'`; do cfy events list -vv $plugin; sleep 5; done
+            git submodule init
+            git submodule update --remote
       - run:
-          name: create iaas secrets
-          description: The user needs to create the secrets.
-          command: |
-
-            cfy secrets create aws_access_key_id -s `echo "${aws_access_key_id}" | base64 --decode`
-            cfy secrets create aws_secret_access_key -s `echo "${aws_secret_access_key}" | base64 --decode`
-            cfy secrets create aws_region_name -s `echo "${aws_region_name}" | base64 --decode`
-            cfy secrets create ec2_region_endpoint -s `echo "${ec2_region_endpoint}" | base64 --decode`
-
-            cfy secrets create azure_subscription_id -s `echo "${azure_subscription_id}" | base64 --decode`
-            cfy secrets create azure_tenant_id -s `echo "${azure_tenant_id}" | base64 --decode`
-            cfy secrets create azure_client_id -s `echo "${azure_client_id}" | base64 --decode`
-            cfy secrets create azure_client_secret -s `echo "${azure_client_secret}" | base64 --decode`
-            cfy secrets create azure_location -s `echo "${azure_location}" | base64 --decode`
-
-            echo "${gcp_credentials}" | base64 --decode > gcp_credentials.json
-
-            cfy secrets create gcp_credentials -f gcp_credentials.json
+          name: update pip
+          command: pip install -U pip
+      - run:
+          name: install cloudify
+          command: pip install -U cloudify==5.0.0 pytest==4.6.3 pyyaml==3.10
+      - run:
+          name: download manager docker image
+          command: wget http://repository.cloudifysource.org/cloudify/5.0.0/ga-release/cloudify-docker-manager-5.0.0.tar
+      - run:
+          name: load docker image
+          command: docker load -i cloudify-docker-manager-5.0.0.tar
+      - run:
+          name: retain space by dumping the tar
+          command: rm cloudify-docker-manager-5.0.0.tar
+      - run:
+          name: show images
+          command: docker images
+      - run:
+          name: start docker container
+          command: docker run --name cfy_manager -d --restart unless-stopped -v /sys/fs/cgroup:/sys/fs/cgroup:ro --tmpfs /run --tmpfs /run/lock --security-opt seccomp:unconfined --cap-add SYS_ADMIN --network host cloudifyplatform/premium-cloudify-manager-aio
+      - run:
+          name: install local project
+          command: pip install https://github.com/cloudify-incubator/cloudify-ecosystem-test/archive/CY-2570-Plugins-Build.zip
```

Comment:
this is fixed for now ?

Reply:
I will update this before merging. I am testing other repos.

Reply:
@EarthmanT I figured as much :)